libappstream-glib/as-release.c

```diff
@@ -1019,6 +1097,17 @@ as_release_node_parse_dep11 (AsRelease *release, GNode *node,
 			}
 			continue;
 		}
+		if (g_strcmp0 (tmp, "Url") == 0) {
```

Comment:
I'm quite sure that's a lower-cased `url` - in the YAML, only the top nodes are upper-cased, every child node is lower-cased all the way.

Reply:
God catch, thanks.